distributions/AlpineLinux.sh

```diff
@@ -0,0 +1,97 @@
+ERROR_EDGE="electrumX can currently be installed only on the edge Version of Alpine Linux"
+grep -q -F "/edge/main" /etc/apk/repositories > /dev/null || _error "${ERROR_EDGE}"
+grep -q -F "/edge/community" /etc/apk/repositories > /dev/null || _error "${ERROR_EDGE}"
+
+. distributions/base.sh
+
+APK="apk --no-cache"
+
+function install_script_dependencies {
+	REPO="http://dl-cdn.alpinelinux.org/alpine/edge/testing"
+	grep -q -F "${REPO}" /etc/apk/repositories || echo "${REPO}" >> /etc/apk/repositories
+	apk update
+	$APK add leveldb
+	$APK add --virtual electrumX-dep openssl wget gcc g++ leveldb-dev
+}
+
+function add_user {
+	adduser -D electrumx
+	id -u electrumx || _error "Could not add user account" 1
+}
+
+function install_python36 {
+	$APK add python3
+	$APK add --virtual electrumX-python python3-dev
+	python3 -m pip install plyvel || _error "Could not install plyvel" 1
+	ln -s $(which python3.6) /usr/local/bin/python3
+}
+
+function install_git {
+	$APK add --virtual electrumX-git git
+}
+
+function install_rocksdb {
+	$APK add rocksdb
+	$APK add --virtual electrumX-db rocksdb-dev
+}
+
+function install_leveldb {
+	$APK add leveldb
+}
+
+function install_init {
+	# init is not required. Alpine is used for containers running the program directly
+	:
+}
+
+function generate_cert {
+	if ! which openssl > /dev/null 2>&1; then
+		_info "OpenSSL not found. Skipping certificates.."
+		return
+	fi
+	_DIR=$(pwd)
+	mkdir -p /etc/electrumx/
+	cd /etc/electrumx
+	echo "[ req ]
```

Comment:
Can you add a comment with the source for this configuration file?

Reply:
Thanks for catching this. I accidentally added a lot of unnecessary stuff :+1: